Super master

siame opened the pull request.

Commit:
Update README

Commit:
Basic file upload to GCS done

Commit:
Change to send UUID instead of filename as result

Commit:
Force client to download files rather than open

Commit:
Change error message

obmarg commented on flask_gae_blobstore.py:
```diff
@@ -62,6 +62,11 @@
+my_default_retry_params = gcs.RetryParams(initial_delay=0.2,
+                                          max_delay=5.0,
+                                          backoff_factor=2,
+                                          max_retry_period=15)
```
Is this actually used anywhere?

obmarg commented on flask_gae_blobstore.py:
```diff
@@ -274,40 +281,40 @@ def validate_file_type(result, accept_file_types=UPLOAD_ACCEPT_FILE_TYPES):
+                            'x-goog-meta-filename': name,
+                            'Content-Disposition': 'attachment; filename={}'
+                                .format(name)
+                        },
```
Content-disposition, write_retry_params & bucket_name seem like parameters that we might want to change without having to change this library.

Commit:
Add option for custom retry params + buckets

User can now specify their own RetryParams from
Google's CloudStorage library, as well as
specify a name for a custom bucket to upload to.

Commit:
Add setup.py as executable
